.github/workflows/sca.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         run: ./dev-tools/check_trailing_spaces.sh
 
       - name: Check - phpstan
-        run: ./dev-tools/vendor/bin/phpstan analyse --error-format=github
+        run: ./dev-tools/vendor/bin/phpstan analyse --ansi
```

Comment:
we can rely on autodetection of error-format, but not of (no-?)ansi ?

Reply:
![Screenshot_2023-12-02-22-36-30-961_com.Slack-edit.jpg](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/1f97f821-d048-4ae0-8c70-409bdbec332c)

Also @ondrejmirtes confirmed that auto-detection is the way to go. When it comes to `--ansi` I've added this because without enforced ansi the output does not have colors which makes it less readable in Github's output.

Reply:
i'm not against autodetection of gha formatter.
i wonder about missing (wrong?) autodetection of ansi